docs/ci-cd-environment/aws-support.md

```diff
@@ -0,0 +1,112 @@
+---
+Description: This guide describes how to set up a fleet of self-hosted agents in AWS
+---
+
+# AWS support
```

Comment:
Can be improved, assuming this is within _Self-hosted agents_ category, I recommend **AWS Auto-Scaling**.

Reply:
Using **AWS Auto-Scaling** might give a wrong impression that this is only if you want autoscaling capabilities, which is not the case. You can use that CDK application even if you only want a statically sized pool of agents

Reply:
OK, makes sense.